defs/bitcoin/gincoin.json

```diff
@@ -0,0 +1,40 @@
+{
+  "coin_name": "Gincoin",
+  "coin_shortcut": "GIN",
+  "coin_label": "GIN",
+  "website": "https://gincoin.io",
+  "github": "https://github.com/gincoin-dev/gincoin-core",
+  "maintainer": "Dragos Badea <dragos.badea@gincoin.io>",
+  "curve_name": "secp256k1",
+  "address_type": 38,
+  "address_type_p2sh": 10,
+  "maxfee_kb": 100000,
+  "minfee_kb": 1000,
+  "signed_message_header": "DarkCoin Signed Message:\n",
```

Comment:
Are you sure Gincoin developers were sloppy and did not change this constant?

Reply:
They were indeed